doc/source/conf.py

```diff
@@ -24,9 +24,12 @@
 author = 'Dominikus Heinzeller, Alexander Richert, Cameron Book'
 
 # The short X.Y version
-version = '1.5'
+#version = '1.5'
```

Comment:
I did. Every six weeks these days I am creating a new release and this would save us half a second ... it doesn't really matter I guess, might as well remove.

Reply:
Removed